posthog/models/web_preaggregated/sql.py

```diff
@@ -285,8 +285,8 @@ def WEB_STATS_INSERT_SQL(
             e.mat_$device_type AS device_type,
             e.mat_$browser AS browser,
             e.mat_$os AS os,
-            e.mat_$viewport_width AS viewport_width,
-            e.mat_$viewport_height AS viewport_height,
+            accurateCastOrNull(e.mat_$viewport_width, 'Int64') AS viewport_width,
```

Comment:
When generating demo data, sometimes it picks the wrong types for these columns. This should have no overhead if the column is the correct type